test/test_core.py

```diff
@@ -1954,10 +1954,8 @@ def test_em_js(self, args, force_c):
 
   @no_wasm2js('WASM_BIGINT is not compatible with wasm2js')
   def test_em_js_i64(self):
-    err = self.expect_fail([EMCC, '-Werror', test_file('core/test_em_js_i64.c')])
+    err = self.expect_fail([EMCC, '-Werror', '-sWASM_BIGINT=0', test_file('core/test_em_js_i64.c')])
     self.assertContained('emcc: error: using 64-bit arguments in EM_JS function without WASM_BIGINT is not yet fully supported: `foo`', err)
-
-    self.set_setting('WASM_BIGINT')
     self.node_args += shared.node_bigint_flags(self.get_nodejs())
```

Comment:
I guess this line should no longer be needed since it will be the default now?

Reply:
The test still still executes the first part without bigint and the second part with it, I just use the command line flag instead of set_setting.